docs/.vuepress/config/sidebar-developer.js

```diff
@@ -201,11 +201,12 @@ const developer = [
         ],
       },
       ['/developer-docs/latest/development/admin-customization', 'Admin panel customization'],
-      ['/developer-docs/latest/development/plugins-extension.html', 'Plugins extension'],
-      ['/developer-docs/latest/development/plugins-development.html', 'Plugins development'],
-      ['/developer-docs/latest/development/custom-fields.html', 'Custom fields'],
-      ['/developer-docs/latest/development/typescript.html', 'TypeScript'],
-      ['/developer-docs/latest/development/providers.html', 'Providers'],
+      ['/developer-docs/latest/development/plugins-extension.md', 'Plugins extension'],
```

Comment:
Out of curiosity since I don't know how the docs are built: why is the extension changing from html -> md?

I notice other docs still use html, and the "Admin panel customization" above doesn't even have an extension.

Reply:
Thanks for noticing, Ben!
Ideally, we should not have any extension (or a `.md` one) — in both cases, the VuePress router would handle the routing.
But Melanie brought to our attention that SEMrush, a tool analyzing SEO, listed such pages as 404, so we must use `.html` everywhere. @StrapiShaun, this has been fixed in [this PR](https://github.com/strapi/documentation/pull/1289/files). Could you please merge your branch with `main` again?

Reply:
fixed now.